example_remoteVerify_test.go

```diff
@@ -0,0 +1,79 @@
+package notation_test
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/notaryproject/notation-core-go/signature/cose"
+	_ "github.com/notaryproject/notation-core-go/signature/jws"
+	"github.com/notaryproject/notation-go"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/internal/mock"
+	"github.com/notaryproject/notation-go/verifier"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation-go/verifier/truststore"
+)
+
+// ExampleRemoteVerify demonstrates how to use notation.Verify to verify
+// signatures of an artifact in the remote registry.
+func Example_remoteVerify() {
+	exampleArtifactReference := "registry.acme-rockets.io/software/net-monitor@sha256:60043cf45eaebc4c0867fea485a039b598f52fd09fd5b07b0b2d2f88fad9d74e"
+
+	// exampleRemoteVerifyOptions is an example of notation.RemoteVerifyOptions.
+	exampleRemoteVerifyOptions := notation.RemoteVerifyOptions{
+		ArtifactReference:    exampleArtifactReference,
+		MaxSignatureAttempts: 50,
+	}
```

Comment:
It seems `exampleArtifactReference` can be merged to `exampleRemoteVerifyOptions`